pages/vi/vi-planetapps.md

```diff
@@ -84,12 +84,12 @@ Take the course, and the last question will ask you to specify any problems or i
   ![Submissions button](images/vi-planet-submissions-button.png)
   ![View Submissions](images/vi-planet-submissions.png)
 
-6. When you provide feedback, try to be as specific as possible, and include screenshots when necessary
+6. When you provide feedback, try to be as specific as possible, and include screenshots when necessary, through your preffered method (Gitter chat is always a great fallback if you are not comfortable with uploading and linking screenshots).
```

Comment:
"preferred"- Please correct this part.

Reply:
I changed it @ChiragAwale , not sure how I missed that during my review. Spelling still wasn't quite right after my first commit to fix it, hopefully it is now, and my apologies for such a simple mistake. I made sure the raw.githack updated to reflect the changes. https://raw.githack.com/johnmdrake1/johnmdrake1.github.io/step4-planet-wordingfix-johnmdrake1/#!pages/vi/vi-planetapps.md